src/redux/tasks/actions/fetch.ts

```diff
@@ -0,0 +1,64 @@
+import { Dispatch } from '../../RootReducer'
+import { API } from '../../../services/http'
+
+import { ITask } from '../types'
+
+export const TASK_SUCCESS = 'TASK_SUCCESS'
+export const TASK_ERROR = 'TASK_ERROR'
+
+export const FETCH_TASK = 'FETCH_TASK'
+export const FETCH_TASK_SUCCESS = 'FETCH_TASK_SUCCESS'
+
+export type Actions = {
+    TASK_ERROR: {    
+        type: typeof TASK_ERROR,
+        error: string,
+    },
+    TASK_SUCCESS: {
+        type: typeof TASK_SUCCESS,
+        successMessage: string,
+    },
+    FETCH_TASK: {  
+        type: typeof FETCH_TASK,
+    },
+    FETCH_TASK_SUCCESS: {    
+        type: typeof FETCH_TASK_SUCCESS,
+        task: ITask,
+    }
+}
+
+export const actionCreators = {
+    // --------------------------------------- //
+    //                    SYNC                 //
+    // --------------------------------------- //
+    taskError: (error: string): Actions[typeof TASK_ERROR] => ({      
+        type: TASK_ERROR,
+        error,
+    }),
+    taskSuccess: (successMessage: string): Actions[typeof TASK_SUCCESS] => ({
```

Comment:
Why I do not get the desired task, instead of a success message ?

Reply:
As the List exemple, taskSuccess is for the message of success, and fetchTaskSuccess get the desired task.